packages/wallet/backend/src/app.ts

```diff
@@ -313,7 +313,29 @@ export class App {
       })
   }
 
+  private async processWMPaymentPointers() {
+    const logger = await this.container.resolve('logger')
+    const paymentPointerService = await this.container.resolve(
+      'paymentPointerService'
+    )
+
+    return paymentPointerService
+      .processWMPaymentPointers()
+      .catch((e) => {
+        logger.error(e)
+        return false
+      })
+      .then((trx) => {
+        if (trx) {
+          process.nextTick(() => this.processWMPaymentPointers())
+        } else {
+          setTimeout(() => this.processWMPaymentPointers(), 5000).unref()
```

Comment:
5 or 10 minutes for the WM worker?

Reply:
both sound reasonable to me, lets leave it to 5